Classes/UIScrollView+InfiniteScroll.h

```diff
@@ -21,9 +21,19 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NS_ENUM(NSInteger, UIScrollViewInfiniteScrollDirection) {
+    UIScrollViewInfiniteScrollDirectionTop      = 0,
```

Comment:
Upward direction is default in this PR. Would be great to maintain the existing behavior and make bottom default. This is better achieved by moving infinite scroll direction to state object and then resetting it to bottom during `-init`.